uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+2. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
+3. Select a test and run it.
+    - Note: Once the test starts, avoid interacting with your machine (e.g. move mouse, use keyboard) to allow the test to navigate the Dev Home app and execute the test.
+
+### How to configure the application settings for a Dev Home release?
+Application settings are a set of configuration properties loaded from `appsettings.*.json` and are available at runtime to the test methods.
+For a developer release of Dev Home, the default configuration from `appsettings.json` are used. For Canary and Prod, a
+separate application settings JSON file (`appsettings.canary.json` or `appsettings.prod.json`) is available
+and its values will overwrite the ones from the default `appsettings.json` file.
```

Comment:
Super Nit: I think we should leave paragraphs (or at least full sentences) in a single line without breaking into multiple lines. Or if we do want to do the breaking manually, to have it be a consistent width